NorthstarDedicatedTest/serverauthentication.cpp

```diff
@@ -480,6 +483,12 @@ CCommand__TokenizeType CCommand__Tokenize;
 
 char CGameClient__ExecuteStringCommandHook(void* self, uint32_t unknown, const char* pCommandString)
 {
+	// Only log clientcommands if the convar `ns_should_log_all_clientcommands` equals 1
+	if (Cvar_ns_should_log_all_clientcommands->GetBool())
+	{
+		spdlog::info("{} with UID {} sent stringcommand {}", (char*)self + 0x16, (char*)self + 0xF500, pCommandString);
```

Comment:
Formatting is kinda odd, doesn't make it super clear where the command starts and ends (?)

I'd recommend something like:
```cpp
spdlog::info("{} (UID: {}) executed command: \"{}\"", (char*)self + 0x16, (char*)self + 0xF500, pCommandString);
```

Reply:
Changed it to suggested version in 034f1f0b63d0c8dd0040f53c558a17509629a52a